### ISSUE TITLE
Cluster not reachable 

### ISSUE THREAD

**Issue by mjudeikis** [Azure/ARO-RP#932]
I have a suspicion if we ran a pod (In the example in a form of DS ) on each node, mount `/tmp` file system and produce big files, it might crash all control plane.

This should not happen as individual pods should be limited in how much space they can consume from `tmpfs`.

If this code/DS is running on the cluster http://git.bytheb.org/cgit/stap.git/
cluster is not usable after 1-2 days.

We need to do some debugging to understand why this is happening.
Potentially in cluster with local-RP, `persist=true` and some similar pod to produce a lot of data

**Comment by m1kola** [comment]
@ehashman correct: It is not related to tmpfs and this can happen with any host mounted dir. I think there is not much we (or upstream) can do to protect from this. The only more or less reasonable thing is to have `/tmp` separate from the root fs: it will make it harder to break the cluster for people who use `/tmp` as a staging/trash directory. But I'm not sure if worth the effort.
